alvr/session/src/settings.rs

```diff
@@ -271,6 +271,18 @@ pub struct ControllersDesc {
     #[schema(min = 0., max = 5., step = 0.1)]
     pub haptics_intensity: f32,
 
+    #[schema(min = 0., max = 1., step = 0.01)]
+    pub haptics_amplitude_curve: f32,
```

Comment:
This one seems pretty `advanced` to me too? Any reason for not marking it as such?

Reply:
I think it is a basic enough option
It increases haptic strength without "clipping"

Reply:
I'm really bad at naming haha.
Would Haptics Boost work better?

Reply:
Haptics strength?

Reply:
There is already haptics intensity though
Would both need to be renamed?

Reply:
Oic. I haven't been 100% keeping track of this... What's the difference between them?

Reply:
I made visualizations of all the options here
https://www.desmos.com/calculator/e19qhia5aw
https://www.desmos.com/calculator/behvrlwbsv
https://www.desmos.com/calculator/uasyojsi4l

Reply:
I can't quite grok how the equations translate to motion, ngl. Maybe I'm just dumb 😆 Is the amplitude curve how it scales the strength of the vibration, with a curve of zero meaning the vibration strength is linear, and 1 meaning all vibration strengths are the same?

Reply:
Yeah

Reply:
Haptics intensity curve, maybe? That said, knowing what it does, I'd tend to agree with @ckiee that this should probably under advanced settings, since I can't imagine most users will care about tweaking the haptics.

Reply:
What about the regular Haptics intensity setting?

Reply:
Haptics strength? I dunno, naming things is hard 😆